Formula/z/zsh-git-prompt.rb

```diff
@@ -1,16 +1,22 @@
 class ZshGitPrompt < Formula
   desc "Informative git prompt for zsh"
-  homepage "https://github.com/olivierverdier/zsh-git-prompt"
-  url "https://github.com/olivierverdier/zsh-git-prompt/archive/refs/tags/v0.5.tar.gz"
-  sha256 "87e5a908369f402e975426ffd61a8800f1c04c0a293f1d4015a6fb1f4408e77d"
+  homepage "https://github.com/zsh-git-prompt/zsh-git-prompt"
+  url "https://github.com/zsh-git-prompt/zsh-git-prompt/archive/refs/tags/v0.6.tar.gz"
+  sha256 "b28a8249797b92b25e3c3156dc99153548a5b0877d2e6aa20be15caa719dcea2"
   license "MIT"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "d4fa3836434d56704bd03f88f3e45557cac6e12cb6e17cc251635ecdcbc431eb"
   end
 
+  uses_from_macos "zsh"
+
   def install
-    prefix.install Dir["*.{sh,py}"]
+    prefix.install Dir["*.sh"] # for now, only zsh.sh
+    prefix.install "shell"
+    prefix.install "python"
+    # could also install "haskell" folder? but why
+    # prefix.install "haskell"
```

Comment:
let's remove this, it would in the github history

```suggestion
```

Reply:
Is that bad? It's meant as a hint for a future developer who maybe wants to use the haskell implementation. But of course I can remove it